lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,336 @@
+      style: SimpleFillSymbolStyle.diagonalCross,
+      color: Colors.green,
+      outline: outlineSymbol,
+    );
+
+    // Create a new point collection for polygon.
+    final polygonBuilder =
+        PolygonBuilder(spatialReference: SpatialReference.wgs84);
+    polygonBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6425,
+        y: 56.0784,
+      ),
+    );
+    polygonBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6430,
+        y: 56.0763,
+      ),
+    );
+    polygonBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6410,
+        y: 56.0759,
+      ),
+    );
+    polygonBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6380,
+        y: 56.0765,
+      ),
+    );
+    polygonBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6380,
+        y: 56.0784,
+      ),
+    );
+    polygonBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6410,
+        y: 56.0786,
+      ),
+    );
+    final polygon = polygonBuilder.toGeometry();
+
+    // Create the graphic with polygon and symbol.
+    final graphic = Graphic(geometry: polygon, symbol: fillSymbol);
+
+    // Add graphic to the graphics overlay.
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createText() async {
+    // Create two text symbols.
+    TextSymbol bassRockTextSymbol = TextSymbol(
+      text: 'Black Rock',
+      color: Colors.blue,
+      size: 10,
+      horizontalAlignment: HorizontalAlignment.left,
+      verticalAlignment: VerticalAlignment.bottom,
+    );
+
+    TextSymbol craigleithTextSymbol = TextSymbol(
+      text: 'Craigleith',
+      color: Colors.blue,
+      size: 10,
+      horizontalAlignment: HorizontalAlignment.right,
+      verticalAlignment: VerticalAlignment.top,
+    );
+
+    // Create two points.
+    ArcGISPoint bassPoint = ArcGISPoint(
+        x: -2.64, y: 56.079, spatialReference: SpatialReference.wgs84,);
+    ArcGISPoint craigleithPoint = ArcGISPoint(
+        x: -2.72, y: 56.076, spatialReference: SpatialReference.wgs84,);
+
+    // Create two graphics from the points and symbols.
+    Graphic bassRockGraphic = Graphic(
+      geometry: bassPoint,
+      symbol: bassRockTextSymbol,
+    );
+    Graphic craigleithGraphic = Graphic(
+      geometry: craigleithPoint,
+      symbol: craigleithTextSymbol,
+    );
+
+    // Add graphics to the graphics overlay.
+    _graphicsOverlay.graphics.add(
+      bassRockGraphic,
+    );
+    _graphicsOverlay.graphics.add(
+      craigleithGraphic,
+    );
+  }
+
+  void _setExtent() async {
+    // Get all of the graphics contained in the graphics overlay.
+    List<Graphic> myGraphicCollection = _graphicsOverlay.graphics;
+
+    // Create a new envelope builder using the same spatial reference as the graphics.
+    EnvelopeBuilder myEnvelopeBuilder = EnvelopeBuilder(
+      spatialReference: SpatialReference.wgs84,
+    );
+
+    // Loop through each graphic in the graphic collection.
+    for (final Graphic graphic in myGraphicCollection) {
```

Comment:
This could just be `for (final graphic in _graphicsOverlay.graphics) {`

It doesn't need to specify `Graphic` and I don't think the `myGraphicCollection` variable is doing anything useful.